migrations/versions/1272e4b54a32_add_user_bitschema_attribute.py

```diff
@@ -0,0 +1,638 @@
+                    "MEXICO_BAJASUR",
+                    "MEXICO_GENERAL",
+                    "NZ",
+                    "NZ_CHAT",
+                    "NAVAJO",
+                    "PRC",
+                    "PST8PDT",
+                    "PACIFIC_APIA",
+                    "PACIFIC_AUCKLAND",
+                    "PACIFIC_BOUGAINVILLE",
+                    "PACIFIC_CHATHAM",
+                    "PACIFIC_CHUUK",
+                    "PACIFIC_EASTER",
+                    "PACIFIC_EFATE",
+                    "PACIFIC_ENDERBURY",
+                    "PACIFIC_FAKAOFO",
+                    "PACIFIC__FIJI",
+                    "PACIFIC_FUNAFUTI",
+                    "PACIFIC_GALAPAGOS",
+                    "PACIFIC_GAMBIER",
+                    "PACIFIC_GUADALCANAL",
+                    "PACIFIC_GUAM",
+                    "PACIFIC_HONOLULU",
+                    "PACIFIC_JOHNSTON",
+                    "PACIFIC_KIRITIMATI",
+                    "PACIFIC_KOSRAE",
+                    "PACIFIC_KWAJALEIN",
+                    "PACIFIC_MAJURO",
+                    "PACIFIC_MARQUESAS",
+                    "PACIFIC_MIDWAY",
+                    "PACIFIC_NAURU",
+                    "PACIFIC_NIUE",
+                    "PACIFIC_NORFOLK",
+                    "PACIFIC_NOUMEA",
+                    "PACIFIC_PAGO_PAGO",
+                    "PACIFIC_PALAU",
+                    "PACIFIC_PITCAIRN",
+                    "PACIFIC_POHNPEI",
+                    "PACIFIC_PONAPE",
+                    "PACIFIC_PORT_MORESBY",
+                    "PACIFIC_RAROTONGA",
+                    "PACIFIC_SAIPAN",
+                    "PACIFIC_SAMOA",
+                    "PACIFIC_TAHITI",
+                    "PACIFIC_TARAWA",
+                    "PACIFIC_TONGATAPU",
+                    "PACIFIC_TRUK",
+                    "PACIFIC_WAKE",
+                    "PACIFIC_WALLIS",
+                    "PACIFIC_YAP",
+                    "POLAND",
+                    "PORTUGAL",
+                    "ROC",
+                    "ROK",
+                    "SINGAPORE",
+                    "TURKEY",
+                    "UCT",
+                    "US_ALASKA",
+                    "US_ALEUTIAN",
+                    "US_ARIZONA",
+                    "US_CENTRAL",
+                    "US_EAST_INDIANA",
+                    "US_EASTERN",
+                    "US_HAWAII",
+                    "US_INDIANA_STARKE",
+                    "US_MICHIGAN",
+                    "US_MOUNTAIN",
+                    "US_PACIFIC",
+                    "US_SAMOA",
+                    "UTC",
+                    "UNIVERSAL",
+                    "W_SU",
+                    "WET",
+                    "ZULU",
+                    name="timezone",
+                ),
+                nullable=True,
+            )
+        )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
These comments from alembic can be removed right?

Reply:
@isabelcosta , done 😉